gpaste-reloaded@feuerfuchs.eu/files/gpaste-reloaded@feuerfuchs.eu/applet.js

```diff
@@ -12,13 +12,21 @@ const SignalManager = imports.misc.signalManager;
 
 let GPaste; // Will be assigned in entry point
 
-const AppletDir                = imports.ui.appletManager.applets[uuid];
-const _                        = AppletDir.__init__._;
```

Comment:
Unlike in #1683, you remove the import for _ altogether for this applet. Is that intentional?

Reply:
Sorry about that, I thought I was targeting [this version](https://github.com/linuxmint/Cinnamon/pull/7349) of the xlet importer, where CJS was importing `__init__` automatically like it used to, but we decided to scrap it.